### PR TITLE
Fixes issue#493

### PR DESCRIPTION
**Pull request by nbkju9b**
Fixes issue #493

### CODE REVIEW

**Comment by nbkju9b** [comment]
hi,

**Review by nbkju9b — COMMENTED** [review]
removed the comma.

**Review by CBID2 — APPROVED** [review]
Well done @nbkju9b! 😁

**Comment by nbkju9b** [comment]
Thanks a ton Christine 😊

**Review by nbkju9b — COMMENTED** [review]
fixed the issue #493 , it is now showing the correct content

File: database/languages/python.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 [
-   
-]
\ No newline at end of file
+    {
+        "name": "Learn Python 3",
+        "description": "Learn the basics of Python 3, one of the most powerful, versatile, and in-demand programming languages today.",
+        "url": "https://www.codecademy.com/learn/learn-python-3",
+        "category": "Other",
```
Comment (rupali-codes) [inline database/languages/python.json:6]: Should be languages

**Review by CBID2 — APPROVED** [review]
Well done @nbkju9b! :)

File: database/languages/python.json (hunk 1 above)
Comment (CBID2) [inline database/languages/python.json:6]:
```suggestion
        "category": "languages",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Sorry I made a mistake. Fix this one

**Review by CBID2 — APPROVED** [review]
Now, we can merge! 😀

**Review by nbkju9b — COMMENTED** [review]
Thanks Rupali and Christine for the quick review and correction on category. Please see if this can be merged now

**Review by ujjawaltyagii — APPROVED** [review]
Looks good to me ! 👍

**Review by rupali-codes — APPROVED** [review]
thanks

**Pull request merged**